addon/components/markdown-to-html.js

```diff
@@ -26,14 +26,23 @@ export default Ember.Component.extend({
       'tasklists',
       'smoothLivePreview'
     );
+    var markdown = this.get('markdown') || '';
+    var source;
+
+    if (markdown instanceof Ember.Handlebars.SafeString) {
+      source = markdown.toHTML();
+    } else if (Array.isArray(markdown)) {
```

Comment:
I don't understand the array unwrapping here, is that for one of the SafeString/htmlSafe APIs?